sdk/eventhub/azure-eventhub/azure/eventhub/_utils.py

```diff
@@ -122,6 +124,35 @@ def set_message_partition_key(message, partition_key):
         message.header = header
 
 
+@contextmanager
+def send_context_manager():
+    span_impl_type = (
+        settings.tracing_implementation()
+    )  # type: Type[AbstractSpan]
+
+    if span_impl_type is not None:
+        with span_impl_type(name="Azure.EventHubs.send") as child:
+            child.kind = SpanKind.CLIENT
+            yield child
+    else:
+        yield None
```

Comment:
Huh, I did not know that you could `yield` `None` in a `@contextmanager` decorated function... but thinking about it, it makes sense given how the `yield`ed value is used.

Reply:
Yep, it just means you get `None` when using the `with ... as variable`